### PR TITLE
crimson: extract formatter for time_point types

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/crimson/common/formatter.cc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#include "formatter.h"
```
Comment (rzarzynski) [inline src/crimson/common/formatter.cc]:
```cpp
// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
// vim: ts=8 sw=2 smarttab
```
at the beginning?

File: src/crimson/common/formatter.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#include <seastar/core/lowres_clock.hh>
```
Comment (rzarzynski) [inline src/crimson/common/formatter.h]: Same here.

**Comment by tchaikov** [comment]
@rzarzynski fixed and repushed.

**Comment by tchaikov** [comment]
```
188/191 Test #183: unittest_rbd_mirror .....................***Exception: Child aborted156.44 sec
```

**Comment by toabctl** [comment]
@tchaikov why are we merging changes where make check failed?

**Pull request merged**